dask/array/reductions.py

```diff
@@ -438,6 +451,14 @@ def max(a, axis=None, keepdims=False, split_every=None, out=None):
     )
 
 
+def chunk_max(x, axis=None, keepdims=None, *args, **kwargs):
+    if x.size == 0:
+        return np.array([-sys.maxsize])
+    else:
+        r = np.max(x, axis=axis, keepdims=keepdims, *args, **kwargs)
+        return r
+
+
```

Comment:
Similar comment here about returning an empty array and then setting `combine=chunk.max`